js/main.js

```diff
@@ -0,0 +1,119 @@
+(function() {
+
+    function loadPics() {
+        //supplied values from flickr
+        var api_key = '74e47a159e15cbcb6139ba9c9df64c13';
+        var secret = 'c73c5dcbac8e4a96';
+        var method = "flickr.photos.search";
+        var user_id = "138698049@N03";
+        var format = '&format=json';
+        var url = "https://api.flickr.com/services/rest"+"?method="+method+"&api_key="+api_key+"&user_id="+user_id+format+'&nojsoncallback=1';
+
+        var xmlhttp = new XMLHttpRequest();
+
+        xmlhttp.onreadystatechange = function () {
+            var farm, server, id, title, secret, picUrl, list, img, items;
```

Comment:
This is ok, but I prefer a separate line and `var` statement for each. Makes it a bit easier to read. :+1: for having them at the top of the function block.